java/code/src/com/redhat/rhn/domain/channel/Channel.hbm.xml

```diff
@@ -685,13 +686,17 @@ PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
 
     <query name="Channel.findCustomChannelsWithRepositories">
         <![CDATA[from com.redhat.rhn.domain.channel.Channel as c
-                where c.org is not null and c.sources is not empty]]>
+                where c.org is not null and c.sources is not empty and c.autoSync is true]]>
     </query>
 
     <query name="Channel.findVendorChannels">
         <![CDATA[from com.redhat.rhn.domain.channel.Channel as c
                  where c.org is null]]>
     </query>
+    <query name="Channel.findVendorChannelsForAutoSync">
+        <![CDATA[from com.redhat.rhn.domain.channel.Channel as c
```

Comment:
We should put this new query outside of the hbm and directly in `ChannelFactor.listVendorChannelsForAutoSync()` as we are going to that anyway as part of [hibernate migration](https://github.com/uyuni-project/uyuni/wiki/Hibernate#migrate-from-hbmxml-to-annotations)